packages/zipkin/index.d.ts

```diff
@@ -272,20 +268,8 @@ declare namespace zipkin {
     letContext<V>(ctx: TraceId, callback: () => V): V;
   }
 
-  namespace sampler {
```

Comment:
dupe huh?

Reply:
Yes I added a declaration for `Sampler` in 95abee9f43165cd30e9066567df260e66a8e1ee1, and pawel’s commit dc62adeb3dc47eafff02abedff50356ba5f8278f, which was rebased on top, also contained `Sampler`.